### DISCUSSION TITLE
Line too long

### DISCUSSION THREAD

**Discussion by christianbraeunlich** [StefanMaron/BusinessCentral.LinterCop#176] [New Rule]
I'd like to avoid the lines of code become too long

Inspired by:
- https://peps.python.org/pep-0008/#maximum-line-length
- https://www.flake8rules.com/rules/E501.html

It would also be very interesting to be able to set the maximum line length in settings.json itself

**Comment by mjmatthiesen** [comment]
You can add a rules in vs code. It's not the same, but this is what I use to control line length, though I ignore it for procedure headers and other pieces I don't like to split, since vs code has massive issues with large files in larger projects.

```

    "editor.rulers": [
        any number that is higher than 80 because 80 is a dumb number to use.
    ],
```
